webapp/src/selectors/index.ts

```diff
@@ -23,10 +23,13 @@ export const enableEditPermissions = (state: GlobalState) => Boolean(collaboraCo
 
 export function makeGetIsCurrentUserFileOwner(): (state: GlobalState, fileInfo: FileInfo) => boolean {
     return createSelector(
+        (_, fileInfo: FileInfo) => fileInfo,
         (state: GlobalState, fileInfo: FileInfo) => getPost(state, fileInfo.post_id || ''),
         (state: GlobalState) => getCurrentUser(state),
-        (post, currentUser) => {
-            return Boolean(post?.user_id === currentUser.id);
+        (fileInfo, post, currentUser) => {
+            // for the existing attachment, user_id is fetched from post
+            // but, for the newly created attachment, user_id is not present in fileinfo,
```

Comment:
This comment is a bit confusing and can be made unambiguous.

For the `fileInfo.user_id === currentUser.id` condition you have added, in which case will the `fileInfo` be used for `user_id`? 
(existing attachment or new attachment)

Reply:
done, updated that.